example_site/src/components/TemplateBrowser.js

```diff
@@ -39,7 +48,7 @@ function Filter({
   setTemplateFilter,
 }) {
   const [isFilterApplied, setIsFilterApplied] = useState(true);
-  const filterNameColor = templateFilter[filterKey]===filter.value ? theBlue : 'black';
+  const filterNameColor = templateFilter[filterKey] === filter.value ? theBlue : 'black';
```

Comment:
I know you just added the spaces back in here, but since we're looking at it can you add a color constant for black like we do for `theBlue`?

Let's adopt a new naming structure here as well - call them `blueColor` and `blackColor`.